data_validation/query_builder/partition_row_builder.py

```diff
@@ -20,7 +20,7 @@
 class PartitionRowBuilder(object):
     def __init__(
         self,
-        partition_key: str,
+        primary_keys: [str],
```

Comment:
@sundar-mudupalli-work, what is the rationale for changing the partition key argument to primary key? I'm not opposed just worried that primary key may be referenced in other paths of the tool

Reply:
We use the primary-key(s) to match corresponding rows between the source and target. When we partition the table, we only compare one partition in the source with the equivalent (corresponding) partition in the target. So ensuring the same set of primary keys fall into the same partition in both the source and target is necessary and sufficient. If the partition-key uniquely identifies the row, then the partition key can be used instead of the primary key. I am going to update the document on this point - the primary key is only used for matching rows and does not need to be a `primary` key in the table schema. When we say primary key, it can be any column (or set of columns) that uniquely identifies the row.

Reply:
change that to sequence of columns - the order is important.